visual-js/.changeset/shy-bananas-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+"@saucelabs/nightwatch-sauce-visual-service": patch
+---
+
+ignore selectors
```

Comment:
Suggestion: Since that will be part of the changelog, that's always better to have a more explanatory comment to explain user what as changed.